lib/utils/pmid.rb

```diff
@@ -20,89 +20,65 @@ def self.pubmed_url(pubmed_id)
 
   private
   def self.url_for_pubmed_id(pubmed_id)
-    "https://www.ncbi.nlm.nih.gov/pubmed/#{pubmed_id}?report=xml&format=text"
+    "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=pubmed&id=#{pubmed_id}&retmode=json&tool=DGIdb&email=help@dgidb.org"
   end
 
   class PubMedResponse
-    attr_reader :xml
-    def initialize(response_body)
-      @xml = Nokogiri::XML(Nokogiri::XML(response_body).text)
+    attr_reader :result
+    def initialize(response_body, pmid)
+      @result = JSON.parse(response_body)['result'][pmid]
     end
 
     def citation
       [first_author, year, article_title, journal].compact.join(', ')
     end
 
     def authors
-      xml.xpath('//AuthorList/Author').to_a.each.with_index(1).map do |author, i|
-        {
-          fore_name: author.xpath('ForeName').text,
-          last_name: author.xpath('LastName').text,
-          author_position: i
-        }
-      end
+      result['authors'].map{|a| a['name']}
     end
 
     def pmc_id
-      xpath_contents_or_nil("//PubmedData/ArticleIdList/ArticleId[@IdType='pmc']")
+      result['articleids'].each do |articles|
```

Comment:
[16] pry(main)> response.pmc_id
=> [{"idtype"=>"pubmed", "idtypen"=>1, "value"=>"1"},
 {"idtype"=>"rid", "idtypen"=>8, "value"=>"1"},
 {"idtype"=>"eid", "idtypen"=>8, "value"=>"1"}]

I feel like we need a return nil at the end of this function, in the case where there is no pmc_id